tools/hidl/hidlcomp.cpp

```diff
@@ -4428,13 +4428,13 @@ void EspMessageInfo::write_esp()
         if (hasMapInfo())
             outf("\tdouble clientVer = ctx ? ctx->getClientVersion() : -1;\n");
         //attributes first
-        int attrCount=0;
+        int attrribCount=0;
```

Comment:
Not sure how renaming these variables is related to fixing the warnings, and this seems to accidentally have a double rr in it (which makes it heard to read).